_episodes/09-pub-quality-figs.md

```diff
@@ -13,13 +13,66 @@ keypoints:
 - "Publication quality figures are often accompanied by a descriptive caption." 
 ---
 
-`ggplot2` customization for publication quality figures
+Publication quality figures are clean and direct data visualizations that
+clearly illustrate your conclusion to the research question. Publication quality
+figures should include only the necessary data and visualization elements to
+convey the desired message, which are organized by a simple plot frame or theme. 
+All visualization elements are clearly labeled and/or included in a legend and
+are described in the accompanying caption for the graphic. These elements
+include the axis, data shape and color, and any statistical elements giving
+analytical context to the data.
```

Comment:
Q: is it worth having a callout somewhere saying that not all figures need to be publication quality?  I.e., it's OK to cut corners and do the "wrong" thing sometimes, and explain when?